draft-ietf-tls-esni.md

```diff
@@ -118,12 +118,37 @@ notation comes from {{RFC8446, Section 3}}.
 
 # Overview
 
+ECH is a protocol extension to TLS1.3 which objective is to encrypt the
+ClientHello to fulfill a number of goals (see {{goals}}), in particular
+hiding the destination and therefore the Server Name Indication (SNI).
```

Comment:
This paragraph is redundant.  It is already covered by the "Introduction" and "Security Considerations" sections.

Reply:
This is correct and I am aware of it. The problem is that I took a very step by step approach because I am new to this work and the group who worked on this for years so I didn't feel right to change too many things. My first priority here was to put a real overview in the section that is called overview.

The current text before the topology section is not an overview and is in fact the trailing text for the topologies section. 

This simply calls for the following proposal to simply remove the line '# Overview' and take the above lines 121-123 and merge them back to the introduction.